README.md

```diff
@@ -113,6 +113,23 @@ The Grafana dashboards page should open for you automatically. If not, navigate
 to `localhost:13300/dashboards` in your browser and begin exploring the
 available dashboards.
 
+### Visualising cbbackupmgr stats files
+Ensure that you have access to either a backup archive or a cbbackupmgr collect zip
+file.
+
+Start Promtimer:
+
+```bash
+promtimer --stats-archive-path <path-to-backup-archive-or-zip>
+```
```

Comment:
Would it be useful (or even meaningful) to be able to specify multiple archives/zips and view them all together?
I don't know if that'd be a common/useful thing to do with backup stats, but if it were they could be separate data sources/separated by labels.

Reply:
As discussed, I don't think this would provide much benefit. That being said, if users would like this feature later on, I'd be more than happy to add it in.